src/main.cpp

```diff
@@ -403,7 +404,7 @@ void create_solar_system()
     auto &stationary = uni.trajectory_create<universe::TrajStationary>(
                                         uni, uni.sat_root());
 
-    for (int i = 0; i < 20; i ++)
+    for (int i = 0; i < 10; i ++)
```

Comment:
magic number?

Reply:
Most of main.cpp is magical. The entire file is like a config, or a simple demonstration of using OSP through a C++ interface. Perhaps a comment explaining the loop is sufficient?